src/core/load-fragment.c

```diff
@@ -2905,6 +2906,79 @@ int config_parse_unset_environ(
         return 0;
 }
 
+int config_parse_headers(
```

Comment:
src/core/ is the place for code touching pid1 itself. This is not be placed there, because it#s specific to journal-remote stuff.